### DISCUSSION TITLE
Remove Sensor

### DISCUSSION THREAD

**Discussion by whitepoll** [telekom-security/tpotce#1614] [Q&A]
Hello everyone.
If I need to remove a sensor how can I do to remove it cleanly while also removing all associated users and configurations? 
thanks

**Comment by t3chn0m4g3** [comment]
In `$HOME/tpotce/.env` you will find the section `LS_WEB_USER`. It contains all users for sensor log transmission as individual base64 encoded strings. Simply remove the base64 encoded string corresponding to your sensor and restart T-Pot.

**Reply by t3chn0m4g3** [reply]
@whitepoll Updated the answer, had the VAR wrong.

**Reply by vasokolov** [reply]
hello everyone!
But how to remove sensor (for example cowrie) from tpot?
I have tried to put enough # into docker-compose.yml but when i start dps, it's signalized me that cowrie did not work. 
Simple to say: what is a right way to exclude unnessesary honeypots from my tpot instance?
